CHANGELOG.md

```diff
@@ -25,6 +25,12 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+# [5.0.4] - 2024-08-29
```

Comment:
Can we use 5.1.0 version here same as per the discussion on  csm-api-client repo

Reply:
Yeah, I had the same thought. But was just sceptical as we are closing to the cut off date and no other changes would go in.

Reply:
There will be a good amount of changes will be there after cfs V2 to V3 migration. Good to go with version update

Reply:
Sure. will go ahead with 5.1.0 itself